std/random.d

```diff
@@ -1299,19 +1373,59 @@ random number sequences every run.
 Returns:
 A single unsigned integer seed value, different on each successive call
 */
-@property uint unpredictableSeed() @trusted nothrow @nogc
+alias unpredictableSeed = unpredictableSeedOf!uint;
+/// ditto
+@property UIntType unpredictableSeedOf(UIntType)() @nogc nothrow @trusted
+if (isUnsigned!UIntType)
 {
-    import core.thread : Thread, getpid, MonoTime;
-    static bool seeded;
-    static MinstdRand0 rand;
-    if (!seeded)
+    version (AnyARC4Random)
+    {
+        // On macOS if we just need 32 bits it is faster to use
```

Comment:
AFAICT is this no longer only macOS, so "/On macOS/d"?

Reply:
I wrote "on macOS" because I've timed this way is faster on macOS but I haven't timed it on other platforms. I *suspect* it's faster on OpenBSD etc. but I don't *know*.

Reply:
Ah my point was just that the `static if` below is `(UIntType.sizeof <= uint.sizeof)`, so you don't distinguish between macOS, but mention it in your motivation which is a bit confusing.

> I suspect it's faster on OpenBSD etc. but I don't know.

I think so too. It's just returning one register value and no looping should be necessary.